string action 
int64 task_id
string status